### PR TITLE
Hebrew Translation

### PR DESCRIPTION
**Pull request by thebeaar**
i added translation into hebrew in another file in lang, named hw_is.json

### CODE REVIEW

**Comment by KingContaria** [comment]
Thanks for the PR! :)
You'll need one more person verifying your translations, please check the newly added translations workflow: https://github.com/KingContaria/seedqueue/issues/66

**Comment by thebeaar** [comment]
> Thanks for the PR! :) You'll need one more person verifying your translations, please check the newly added translations workflow: #66
"@Boombaloom verified the translations in the discord translation section under Hebrew.

File: src/main/resources/assets/seedqueue/lang/hw_is.json [hunk 1]
```diff
@@ -0,0 +1,123 @@
+
```
Comment (KingContaria) [inline src/main/resources/assets/seedqueue/lang/hw_is.json:1]: There is a missing "{" here to start the json

File: src/main/resources/assets/seedqueue/lang/he_il.json [hunk 2]
```diff
@@ -0,0 +1,122 @@
+{
+  "seedqueue.menu.clearing": "מנקה... ",
```
Comment (KingContaria) [inline src/main/resources/assets/seedqueue/lang/he_il.json:2]: Could you mention that it's SeedQueue that's being cleared, I want it to be very clear to the user what mod is showing a screen

**Review by KingContaria — APPROVED** [review]
Checked, looks good to me

**Comment by KingContaria** [comment]
With 1.2 there is a couple of new/changed translation keys. I want to release this version fairly soon, if you don't have the time to make the edits i can do the renaming and merge the translations, new translations can be added later.

Renamed:
- "speedrunapi.config.seedqueue.option.backgroundPreviews" -> "speedrunapi.config.seedqueue.option.preparingPreviews"
- "speedrunapi.config.seedqueue.option.backgroundPreviews.description" -> "speedrunapi.config.seedqueue.option.preparingPreviews.description"

Added:
- "speedrunapi.config.seedqueue.option.preparingPreviews.value.-1": "Auto"
- "speedrunapi.config.seedqueue.option.resumeOnFilledQueue": "Resume On Filled Queue"
- "speedrunapi.config.seedqueue.option.resumeOnFilledQueue.description": "Whether to resume generating seeds after the queue is filled.\nThis overrides the Max World Generation % option after the queue capacity is filled."
- "seedqueue.key.scheduleJoin": "Schedule Instance"
- "seedqueue.key.scheduleAll": "Schedule All"

**Comment by KingContaria** [comment]
Thank you!
And also... 😅 

Sorry for this, one more update with 4 new translations was added. Like I said before, absolutely no pressure, I will still merge the translations without the updates if you dont have time. :)

Added:
- "speedrunapi.config.seedqueue.option.chunkMapFreezing": "Chunkmap Freezing"
- "speedrunapi.config.seedqueue.option.chunkMapFreezing.description": "Freezes the chunkmap after the set milliseconds. Useful for the \"Village Resetting\" strategy. Only applies to worlds on the wall that aren't locked."
- "speedrunapi.config.seedqueue.option.chunkMapFreezing.value": "%sms"
- "speedrunapi.config.seedqueue.option.chunkMapFreezing.value.-1": "OFF"

**Comment by KingContaria** [comment]
It seems you accidentally reverted the fix for `"seedqueue.menu.clearing"` mentioning SeedQueue
You also added a linebreak (`\n`) to "seedqueue.menu.benchmark.result", which isnt supported

I also forgot one additional translation that was added by the Chunkmap Freezing update:
- "speedrunapi.config.seedqueue.category.misc": "Misc. Settings"

**Pull request merged**